samples/getting-started/azure-quantum/Getting started with Qiskit and IonQ on Azure Quantum.ipynb

```diff
@@ -0,0 +1,652 @@
+       "«c: 3/══════════════════════════════════════════════════════╩═══════╩══╩═\n",
+       "«                                                           0       1  2 </pre>"
+      ],
+      "text/plain": [
+       "     ┌───┐                                                            »\n",
+       "q_0: ┤ H ├─■──────────────────────────────────────────────────────────»\n",
+       "     ├───┤ │                                                          »\n",
+       "q_1: ┤ H ├─┼─────────■─────────■──────────────────────────────────────»\n",
+       "     ├───┤ │         │         │                                      »\n",
+       "q_2: ┤ H ├─┼─────────┼─────────┼─────────■─────────■─────────■────────»\n",
+       "     ├───┤ │P(2π/3)  │P(2π/3)  │P(2π/3)  │P(2π/3)  │P(2π/3)  │P(2π/3) »\n",
+       "q_3: ┤ X ├─■─────────■─────────■─────────■─────────■─────────■────────»\n",
+       "     └───┘                                                            »\n",
+       "c: 3/═════════════════════════════════════════════════════════════════»\n",
+       "                                                                      »\n",
+       "«                  ┌───┐                                   ┌─┐           \n",
+       "«q_0: ───────────X─┤ H ├─■──────────────■──────────────────┤M├───────────\n",
+       "«                │ └───┘ │P(-π/2) ┌───┐ │                  └╥┘     ┌─┐   \n",
+       "«q_1: ───────────┼───────■────────┤ H ├─┼─────────■─────────╫──────┤M├───\n",
+       "«                │                └───┘ │P(-π/4)  │P(-π/2)  ║ ┌───┐└╥┘┌─┐\n",
+       "«q_2: ─■─────────X──────────────────────■─────────■─────────╫─┤ H ├─╫─┤M├\n",
+       "«      │P(2π/3)                                             ║ └───┘ ║ └╥┘\n",
+       "«q_3: ─■────────────────────────────────────────────────────╫───────╫──╫─\n",
+       "«                                                           ║       ║  ║ \n",
+       "«c: 3/══════════════════════════════════════════════════════╩═══════╩══╩═\n",
+       "«                                                           0       1  2 "
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Create and set up circuit\n",
+    "qpe2 = QuantumCircuit(4, 3)\n",
+    "\n",
+    "# Apply H-Gates to counting qubits:\n",
+    "for qubit in range(3):\n",
+    "    qpe2.h(qubit)\n",
+    "\n",
+    "# Prepare our eigenstate |psi>:\n",
+    "qpe2.x(3)\n",
+    "\n",
+    "# Do the controlled-U operations:\n",
+    "angle = 2*math.pi/3\n",
+    "repetitions = 1\n",
+    "for counting_qubit in range(3):\n",
+    "    for i in range(repetitions):\n",
+    "        qpe2.cp(angle, counting_qubit, 3);\n",
+    "    repetitions *= 2\n",
+    "\n",
+    "# Do the inverse QFT:\n",
+    "qft_dagger(qpe2, 3)\n",
+    "\n",
+    "# Measure of course!\n",
+    "for n in range(3):\n",
+    "    qpe2.measure(n,n)\n",
```

Comment:
It's odd to use QPE and immediately measure the output of the register, as QPE is used over iterative PE mainly when the output needs to be coherent for use in further quantum processing; as per suggestion above, linking to something about iterative PE may help put this in context, perhaps, as being somewhat a contrived example for the purpose of showing integration?

Reply:
I ended up taking this code out.